gettsim/benefits/arbeitslosengeld.py

```diff
@@ -14,7 +14,7 @@ def ui(
 
     """
 
-    alg_entgelt = _alg_entgelt(
+    alg_entgelt = proxy_net_wage_last_year(
         person,
```

Comment:
I changed the interface of the function to give the social security contribution separate. Also I added a parameter to elterngeld. We do not check for private insurance in here yet?! Also not for arbeitslosengeld. Should we change that?

Reply:
Good point. I don't know how they are treated. Do self-employed get Elterngeld? These lump-sum values are supposed to get a fictive net wage, which is what Elterngeld is supposed to replace to some extent.

Reply:
Yes, but for them the last calendar year matters, not the 12 months before birth. I only know because having some solar panels on your roof makes turns you into a business owner by that rule... 

So we can just take last year's tax statement, will need to figure out how income tax classes are handled in couples.